style/buttons.less

```diff
@@ -0,0 +1,20 @@
+@btn-border-radius: 4px;
+@primary-background: #f47642;
+@primary-hover-background: #fa804d;
+
+// Override the default text color from bootstrap-material-design
+.btn-primary,
+.btn-success { color: @white !important; }
```

Comment:
Why is the `!important` necessary here? Either:

1. `bootstrap-material-design`'s selector is more specific *or*
2. `@import 'bootstrap-material-design';` occurs after this file

If 1. then this selector should probably be as specific as the original (unless it's really ugly). If 2. then the import should be moved up.